src/common/docker-compose/index.ts

```diff
@@ -31,7 +31,7 @@ export const generate = async (dependency_manager: LocalDependencyManager): Prom
   for (const node of dependency_manager.graph.nodes) {
     if (node instanceof GatewayNode) {
       compose.services[node.normalized_ref] = {
-        image: 'jwilder/nginx-proxy',
+        image: 'registry.architect.io/architect-nginx/proxy:latest',
```

Comment:
Just thought of this - can I pull this image without being logged in? We should still be able to do a local deploy without having an architect account.

Reply:
Works on my machine

Reply:
Might be nice to make the validation endpoint open though. That currently won't work but I can't think of a reason to protect it

Reply:
Yeah I mentioned that to Dan. @dan-barrett can you make public or think of a reason not to?